kolibri/core/discovery/utils/network/client.py

```diff
@@ -241,3 +245,12 @@ def connect(self, raise_if_unavailable=True):  # noqa: C901
             return False
 
         return True
+
+
+def get_user_agent():
+    # get requests default user agent
+    s = requests.Session()
+    requests_user_agent = s.headers['User-Agent']
+
+    # updated useragent format: User-Agent: <product>/<product-version> <requests user agent>
+    return "Kolibri/{0} {1}".format(kolibri.__version__, requests_user_agent)
```

Comment:
Looks like you can use a similar approach for obtaining the python `requests` version. I appreciate what you're thinking here, but for simplicity, I think we can just formulate the python `requests` header ourselves.

So given this:
```
>>> import requests
>>> requests.__version__
'2.27.1'
```
I think having it like this is just fine:
```python
"Kolibri/{0} python-requests/{1}".format(kolibri.__version__, requests.__version__)

Reply:
Thanks for checking! this is indeed simpler, I pushed an update.

Would you prefer if I inline `get_user_agent()` as it's just one expression now?

Reply:
I think keeping it as a separate function is just fine. It makes it possible to runtime-patch if some other Kolibri usage wants to customize it. Thanks!